templates/PlayFab_Api.cpp.ejs

```diff
@@ -1,6 +1,7 @@
 #include <stdafx.h>
 
 <%- getApiDefine(api) %>
+#ifndef PLAYFAB_DISABLE_STATIC_API
```

Comment:
>PLAYFAB_DISABLE_STATIC_API [](start = 8, length = 26)

This is not defined at any project level or header so I assume we are not testing when this is defined? do we normally test preproccessor defines?

Reply:
It was an unrelated idea-spark I had.
Since we don't define it, it is meaningless for now... but it could be a real thing soon enough...

---
In reply to: [343390440](https://github.com/PlayFab/XPlatCoreTemplate/pull/39#discussion_r343390440) [](ancestors = 343390440)

Reply:
unrelated idea-sparks that don't do anything and aren't tested sounds like something you don't want to check into the SDK. It smells like the beginning of something similar to the `verticalName` issue. It's code that's checked in, untested, and unused.

Reply:
Quite right, removing now.

---
In reply to: [343391841](https://github.com/PlayFab/XPlatCoreTemplate/pull/39#discussion_r343391841) [](ancestors = 343391841)